com.unity.visualeffectgraph/Editor/Models/Contexts/VFXSRPSubOutput.cs

```diff
@@ -65,9 +65,9 @@ public virtual string GetRenderQueueStr()
                     throw new NotImplementedException("Unknown blend mode");
             }
 
-            int rawMaterialOffset = owner.GetMaterialOffset();
-            int materialOffset = Mathf.Clamp(rawMaterialOffset, -50, +50);
-            return baseRenderQueue + materialOffset.ToString("+#;-#;+0");
+            int rawMaterialSortingPriority = owner.GetMaterialSortingPriority();
+            int queueOffset = Mathf.Clamp(rawMaterialSortingPriority, -50, +50);
```

Comment:
We have this const 50 in several places:
- Here
- On BaseShaderGUI.cs line 181
- HDRenderQueue.cs line 110

I do not know if this will change in the future for 100 for instance. I think we should put this constant somewhere on Core. Maybe CoreUtils?

Reply:
The -50/+50 range comes from builtin renderer but I agree, it would be appropriate to factorize this and this PR is a good opportunity. I will take a look at this `CoreUtils` and send you a proposal of the constant naming & location.

Reply:
Made a proposition, I'm backuping here just in case I'm reverting my local change
[Factorization_of_Range.zip](https://github.com/Unity-Technologies/Graphics/files/7258376/Factorization_of_Range.zip)

Reply:
Since it would require to add Core in vfx asmdef, better to think twice doing this, I will do it in an independent branch.

Reply:
I've added a task to our backlog to unify this constant
https://jira.unity3d.com/browse/XPIPELINE-373